verdictd/README.md

```diff
@@ -0,0 +1,56 @@
+# Verdictd
+
+## Introduction
+
+Verdictd is a server(as RATS shelterd) which creates a m-TLS(Mutal Transport Layer Security) connection with Attestation Agent via remote attestation (aka "[RA-TLS](https://raw.githubusercontent.com/cloud-security-research/sgx-ra-tls/master/whitepaper.pdf)").
```

Comment:
`an m-TLS`

Reply:
Please add the link about `Attestation Agent`

Reply:
Attestation Agent's codes aren't checked in. I will update it after the codes are commited.